client-server/phoebe-server

```diff
@@ -595,7 +589,8 @@ def new_bundle(type):
             b = getattr(phoebe, constructor)(**kwargs)
         except Exception as err:
             return _get_response({'success': False, 'error': str(err)}, api=True)
-            if app._debug: raise
+            if app._debug:
+                raise
```

Comment:
can we be more targeted with this raise?

Reply:
how so?

Reply:
by raising a specific exception?

Reply:
oh, this just raises the same exception that triggered the except

Reply:
yep, I'm well aware; I just thought further context might be warranted. But maybe not.